matplotcheck/base.py

```diff
@@ -732,15 +794,25 @@ def assert_line(
         m="Expected line not displayed",
         m2="Line does not cover data set",
     ):
-        """Asserts that there exists a line on Axes ax with slope slope_exp and y intercept intercept_exp and goes at least from x coordinate min_val to x coordinate max_val
+        """Asserts that there exists a line on Axes `ax` with slope `slope_exp` and y-intercept `intercept_exp` and goes at least from x coordinate `min_val` to x coordinate `max_val`
```

Comment:
i think these all need to be double ticks rather than single for RST to work properly. please check when you build the docs.